obspy/io/nordic/tests/test_nordic.py

```diff
@@ -456,8 +456,24 @@ def test_read_picks_across_day_end(self):
             self.assertTrue(pick.time in pick_times)
         self.assertEqual(event_2.origins[0].time, event.origins[0].time)
 
-
-def test_similarity(event_1, event_2):
+    def test_distance_conversion(self):
+        """Check that distances are converted properly."""
```

Comment:
Can you please use 
```
def ...():
    """
    TEXT
    """
    ...
```

as a docstring style to be consistent with the rest of ObsPy.